src/httpserver/http_utils.hpp

```diff
@@ -235,8 +245,12 @@ class http_utils {
      static const char* application_octet_stream;
      static const char* text_plain;
 
+     static const char* upload_filename_template;
+
      static std::vector<std::string> tokenize_url(const std::string&, const char separator = '/');
      static std::string standardize_url(const std::string&);
+
+     static std::string generate_random_upload_filename(std::string &directory);
```

Comment:
cpplint warns here:

    src/httpserver/http_utils.hpp:253:  Is this a non-const reference? If so, make const or use a pointer: std::string &directory  [runtime/references] [2]

Reply:
Is const now (input and output)
And yes, I will use cpplint before pushing in the future.